src/main/java/io/outfoxx/swiftpoet/EnumerationCaseSpec.kt

```diff
@@ -0,0 +1,76 @@
+package io.outfoxx.swiftpoet
+
+import java.lang.IllegalStateException
```

Comment:
Can be removed (in favor of implicitly imported `kotlin.ISE`)

Reply:
IntelliJ and I argue about this all the time

Reply:
BTW, Good catch!